etp-front/src/pages/kayttaja/kayttaja-form.svelte

```diff
@@ -38,12 +38,18 @@
 
   $: isValidForm = Validation.isValidForm(schema);
 
-  $: isPaakayttaja = Kayttajat.isPaakayttaja(whoami);
+  $: isCurrentUserPaakayttaja = Kayttajat.isPaakayttaja(whoami);
   $: isOwnSettings = R.eqProps('id', kayttaja, whoami);
 
+  $: isEditedUserPaakayttaja = Maybe.fold(
+    false,
+    Kayttajat.isPaakayttajaRole,
+    kayttaja.rooli
+  );
+
```

Comment:
Kun tämän luin ensimmäisen kerran, niin tuli väärä kuva. En nyt keksi kunnolla kyllä hyvää nimeä, mutta jokin `isEditingTargetPaakayttaja`, `isPaakayttajaSettings` tms voisi olla selkeämpi. 🤔 

Varmaan myös ihan hyvä näin.